packages/ui/react-ui-gameboard/src/components/Chessboard/Chessboard.tsx

```diff
@@ -99,7 +98,7 @@ const ChessboardComponent = forwardRef<HTMLDivElement, ChessboardProps>(
     }, [grid, model?.pieces.value, promoting]);
 
     return (
-      <div ref={targetRef} tabIndex={-1} className={mx('relative outline-none', classNames)}>
+      <div ref={targetRef} tabIndex={0} className={mx('relative outline-none', classNames)}>
```

Comment:
Since this is itself focusable, it needs a focus ring, probably `dx-focus-ring-inset` unless this has spacing around it.